understanding/20/images-of-text.html

```diff
@@ -130,139 +120,54 @@ <h2>Examples of Images of Text</h2>
          <dt>Customizable font settings in images of text</dt>
          <dd>A Web site allows users to specify font settings and all images of text on the site
                are then provided based on those settings.</dd>
+         <dt>The text replicates an image of text</dt>
```

Comment:
This addresses @detlevhfischer concern
```suggestion
         <dt>The same information is given in text and an image of text</dt>
```